app.js

```diff
@@ -70,6 +70,25 @@ app.get('/rest/:id', function(req, res){
   });
 });
 
- // var server = app.listen(3000, function(){
- //   console.log('Magic is happening on port 3000')
- // });
\ No newline at end of file
+# This file is maintained by a script running via a cronjob
```

Comment:
The code that maintains this file is located in `/home/time/diningin-time-tracker/` on our server box. This code currently lives in it's own(currently private) repo on github but can be made public or added to this repo if necessary.